CONTRIBUTING.md

```diff
@@ -1,6 +1,46 @@
-# Development Guide
+# Contributing
 
-Logstash output plugin for delivering log to Sumo Logic cloud service through HTTP source.
+## Set up development environment
+
+You can use the preconfigured Vagrant virtual machine for development
+It includes everything that is needed to start coding.
+See [./vagrant/README.md](./vagrant/README.md).
+
+If you don't want to or cannot use Vagrant, you need to install the following dependencies:
+
+- [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index.html) as a prerequisite to JRuby,
+- [JRuby](https://www.jruby.org/),
+- [Bundler](https://bundler.io/),
+- optionally [Docker](https://docs.docker.com/get-docker/), if you want to build and run container images.
+
+When your machine is ready (either in Vagrant or not), run this in the root directory of this repository:
+
+```sh
+bundle install
+```
+
+## Running tests
+
+Some of the tests try to send actual data to an actual Sumo Logic account.
+To run those tests, you need to make `sumo_url` environment variable available.
+If the `sumo_url` environment variable is not present, the tests reaching Sumo Logic will be skipped.
```

Comment:
sounds like case for receiver-mock :thinking:

Reply:
Yes, very much. Pull requests are very welcome :D

Reply:
Could you create issue for that? :sweat:

Reply:
https://github.com/SumoLogic/logstash-output-sumologic/issues/69